geometric_shapes.repos

```diff
@@ -1,8 +1,4 @@
 repositories:
-  geometric_shapes:
-    type: git
-    url: https://github.com/ros-planning/geometric_shapes
-    version: ros2
   random_numbers:
```

Comment:
`random_numbers` still isn't released?

Reply:
no, sadly, it is on the things we have planned to do over the next couple weeks though.